test/brady_test.exs

```diff
@@ -62,4 +62,22 @@ defmodule BradyTest do
       assert Brady.body_class(conn) == "admin-user user user-show"
     end
   end
+
+  describe "inline_svg" do
+    test "it renders an html safe version of the SVG" do
+     assert Brady.inline_svg("test", class: "foo", "data-role": "bar") == test_svg
+    end
+
+    test "it returns a runtime error if SVG is not found" do
+      message = "No SVG found at test/support/svg/non_existant.svg"
+      assert_raise RuntimeError, message, fn ->
+        Brady.inline_svg("non_existant")
+      end
+    end
+
+    def test_svg(options \\ []) do
+      {:safe,
+       ~s(<svg class=\"foo\" data-role=\"bar\" height=\"100\" width=\"100\"><desc>This is a test svg</desc><circle cx=\"50\" cy=\"50\" r=\"40\" stroke=\"black\" stroke-width=\"3\" fill=\"red\"></circle></svg>)}
```

Comment:
I believe the intent for the sigil would be to allow 
`~s(<svg class="foo" data-role="bar" height="100" width="100"><desc>This is a test svg</desc><circle cx="50" cy="50" r="40" stroke="black" stroke-width="3" fill="red"></circle></svg>)`
instead of escaping the quotes?